cli/src/cli_util.rs

```diff
@@ -1017,9 +1017,20 @@ impl WorkspaceCommandHelper {
                 let mut iter = [commit0, commit1].into_iter().chain(iter);
                 let commits: Vec<_> = iter.by_ref().take(5).try_collect()?;
                 let elided = iter.next().is_some();
-                let hint = if let RevsetExpression::CommitRef(RevsetCommitRef::Symbol(
-                    branch_name,
-                )) = revset_expression.as_ref()
+                let hint = if commits[0].change_id() == commits[1].change_id() {
+                    // Separate hint if there's commits with same change id
```

Comment:
Same comments about backticks and reusing `{commits}{ellipsis}` as in the parent commit